src/components/Offer.vue

```diff
@@ -94,7 +94,7 @@ export default {
         if (this.offer === '') throw new Error(this.$t('emptyField'))
 
         const res = await callApi(
-          this.$q.localStorage.getItem('server') + 'request/helper',
+          this.$q.localStorage.getItem('server') + '/api/v1/' + 'request/helper',
```

Comment:
```suggestion
          this.$q.localStorage.getItem('server') + '/api/v1/request/helper',
```
Would you like to combine the two strings?

Reply:
Good catch. 👍